test/src/DistortableCollectionSpec.js

```diff
@@ -68,7 +68,7 @@ describe("L.DistortableCollection", function () {
     });
   });
 
-  describe("_toggleMultiSelect", function () {
+  describe("#_toggleMultiSelect", function () {
     it("Should allow selection of multiple images on command + click", function() {
       simulateCommandMousedown(overlay.getElement());
       simulateCommandMousedown(overlay2.getElement());
```

Comment:
Nice! This can be a really useful utility fn. for the devs that build on this. Can we document this (`simulateCommandMousedown`) a bit?

Reply:
Yes I was thinking that too! I initially thought to maybe put it in `specHelper.js` but I am new to testing and had no idea if this is where it belonged or how to correctly place it there. Anyway this is from a past commit, all I did here was add the '#'. I'm going to open a separate issue for the `simulateCommandMousedown` function right now and mention you!

Reply:
Ok opened #213 !